src/components/ReportActionItemIOUPreview.js

```diff
@@ -0,0 +1,135 @@
+import React from 'react';
+import {View, TouchableOpacity} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+import lodashGet from 'lodash/get';
+import ONYXKEYS from '../ONYXKEYS';
+import ReportActionItemIOUQuote from './ReportActionItemIOUQuote';
+import ReportActionPropTypes from '../pages/home/report/ReportActionPropTypes';
+import Text from './Text';
+import MultipleAvatars from './MultipleAvatars';
+import styles from '../styles/styles';
+
+const propTypes = {
+    // All the data of the action
+    action: PropTypes.shape(ReportActionPropTypes).isRequired,
+
+    // Is this the most recent IOU Action?
+    isMostRecentIOUReportAction: PropTypes.bool.isRequired,
+
+    // The report currently being looked at
+    report: PropTypes.shape({
+
+        // IOU report ID associated with current report
+        iouReportID: PropTypes.number,
+
+        // Whether there is an outstanding amount in IOU
+        hasOutstandingIOU: PropTypes.bool,
+    }).isRequired,
+
+    /* --- Onyx Props --- */
+    // Active IOU Report for current report
+    iou: PropTypes.shape({
+        // Email address of the manager in this iou report
+        managerEmail: PropTypes.string,
+
+        // Email address of the creator of this iou report
+        ownerEmail: PropTypes.string,
+
+        // Outstanding amount of this transaction
+        cachedTotal: PropTypes.string,
+    }),
+
+    // All of the personal details for everyone
+    personalDetails: PropTypes.objectOf(PropTypes.shape({
+
+        // This is either the user's full name, or their login if full name is an empty string
+        displayName: PropTypes.string.isRequired,
+    })).isRequired,
+
+    // Session info for the currently logged in user.
+    session: PropTypes.shape({
+        // Currently logged in user email
+        email: PropTypes.string,
+    }).isRequired,
+};
+
+const defaultProps = {
+    iou: {},
+};
+
+const ReportActionItemIOUPreview = ({
+    action,
+    isMostRecentIOUReportAction,
+    report,
+    iou,
+    personalDetails,
+    session,
+}) => {
+    const managerName = lodashGet(personalDetails, [iou.managerEmail, 'displayName'], iou.managerEmail);
```

Comment:
Not sure where the best place for this would be, but could you add a small comment that manager email is the current owner of the report? So that the `iou.managerEmail === sessionEmail` makes more sense?

Reply:
I added comment and used `Str.removeSMSDomain` here.